btk/draw_blends.py

```diff
@@ -61,6 +62,33 @@ def get_size(pixel_scale, catalog, i_obs_cond):
     return Column(size, name="size")
 
 
+def make_wcs(pix, shape, center_pix=None, center_sky=None, naxis=2):
```

Comment:
maybe we can move this function to either `create_observing_generator` or `survey` ?

Reply:
It's a possibility, all the information needed is available there, we would just need to extract the wcs from obs_cond at this point in the code.

Reply:
Looking at this again: 
```

        wcs = make_wcs(
            pixel_scale=Args.pixel_scale,
            center_pix=obs_cond[j].center_pix,
            center_sky=obs_cond[j].center_sky,
            projection=obs_cond[j].projection,
            shape=(stamp_size, stamp_size),
        )
```
you are not using any information from the catalog/stamp/galsim Image in creating WCS. Do you think this might change ? if not, I would suggest creating the WCS in the `create_observing_generator.py`

Reply:
No, I do not think we will need the galsim information or any information from the catalog. I can move it to the create_observing_generator if you prefer.

Reply:
Yes, that would be great, thanks!

Reply:
Maybe it would make more sense to create it directly in the survey if we go that way ?